app/controllers/users_controller.rb

```diff
@@ -0,0 +1,15 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+    if @user
```

Comment:
Nên viết code theo dạng sau em nhé, với nội dung message thì nên phản ánh được nội dung. vidu: "user_not_found", "not_found"
```suggestion
    return if @user
    flash[:danger] = t("welcom")
    redirect_to home_path
```

Reply:
comment này chưa fix nữa